apps/console/src/features/api-resources/pages/api-resources.tsx

```diff
@@ -382,7 +382,7 @@ const APIResourcesPage: FunctionComponent<APIResourcesPageInterface> = (
                         data-componentid={ `${ componentId }-list-advanced-search` }
                     />
                 ) }
-                showTopActionPanel={ true }
+                showTopActionPanel={ !(!searchQuery && apiResourcesList?.length === 0) }
```

Comment:
Shall we expand this condition for better readability?

```suggestion
                showTopActionPanel={ searchQuery || apiResourcesList?.length > 0) }
```

Reply:
Addressed with 65e094e9ac8666c568e1e69afa5d63af14247da5